.vscode/tasks.json

```diff
@@ -16,6 +16,132 @@
+        "ESBuild-neovim-registry",
+        "ESBuild-cursorless-neovim",
+        "Build test harness"
+      ],
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-neovim-registry",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/neovim-registry",
+      "dependsOn": [],
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-cursorless-neovim",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/cursorless-neovim",
+      "dependsOn": [
+        "ESBuild-common",
+        "ESBuild-cursorless-engine",
+        "ESBuild-neovim-common"
+      ],
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-common",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/common",
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-cursorless-engine",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/cursorless-engine",
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-neovim-common",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/neovim-common",
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "neovim logs",
+      "type": "shell",
+      "windows": {
+        "command": "powershell",
+        "args": [
+          "(New-Object -ComObject WScript.Shell).Run(\"\"\"powershell Get-Content '${workspaceFolder}/packages/cursorless-neovim/out/nvim_node.log' -wait -tail 30\"\"\", 1, $false)"
+        ]
+      },
+      "group": "build"
+    },
+    {
+      "label": "neovim start",
+      "type": "process",
+      "windows": {
+        "command": "powershell",
+        "args": [
+          "(New-Object -ComObject WScript.Shell).Run(\"\"\"nvim\"\"\", 1, $false)"
+        ]
+      },
+      "group": "build",
+      "options": {
+        "env": {
+          "CURSORLESS_REPO_ROOT": "${workspaceFolder}",
+          "NVIM_NODE_HOST_DEBUG": "1",
+          "NVIM_NODE_LOG_FILE": "${workspaceFolder}/packages/cursorless-neovim/out/nvim_node.log",
+          "NVIM_NODE_LOG_LEVEL": "info",
+          "CURSORLESS_MODE": "development"
+        }
+      }
+    },
+    {
+      "label": "neovim test start",
+      "type": "process",
+      "windows": {
+        "command": "powershell",
+        "args": [
+          "(New-Object -ComObject WScript.Shell).Run(\"\"\"nvim\"\"\", 1, $false)"
+        ]
+      },
+      "group": "build",
+      "options": {
+        "env": {
+          "CURSORLESS_REPO_ROOT": "${workspaceFolder}",
+          "NVIM_NODE_HOST_DEBUG": "1",
+          "NVIM_NODE_LOG_FILE": "${workspaceFolder}/packages/cursorless-neovim/out/nvim_node.log",
```

Comment:
not sure bout putting logs in `out` folder; usually that contains the artifacts that should actually go into the extension. Not a showstopper tho if it's going to be a hassle to do something else; can mention in issue describing better logging approach that we already know we want

Reply:
I created a git issue to track this improvement